server/documents/elements/label.html.eco

```diff
@@ -333,16 +353,89 @@ themes      : ['Default', 'GitHub']
   <div class="example" data-use-content="true" data-class="icon">
     <h4 class="ui header">Icon</h4>
     <p>A label can include an icon</p>
-    <a class="ui label">
+    <div class="ui label">
       <i class="mail icon"></i>
       Mail
-    </a>
-    <a class="ui label">
-      Tag
-      <i class="delete icon"></i>
-    </a>
+    </div>
+    <div class="ui label">
+      <i class="checkmark icon"></i>
+      Test
+    </div>
+    <div class="ui label">
+      <i class="dog icon"></i>
+      Dog
+    </div>
+    <div class="ui label">
+      <i class="cat icon"></i>
+      Cat
+    </div>
   </div>
+  
+  <div class="another example" data-use-content="true">
+    <div class="ignored ui info message">Icons can be placed in the right side of label.</div>
```

Comment:
I think it should read `Icons can be placed to the right inside a label` 
@hammy2899 ? What does the native Speaker say? :wink:

Reply:
I agree with @lubber-de, they are placed "to" the right and they are placed "inside" the label.

Reply:
Fixed in https://github.com/fomantic/Fomantic-UI-Docs/pull/72/commits/56d1d1730bb8970be59d43816802f0bf4961b36b